etna/prediction_intervals/base.py

```diff
@@ -0,0 +1,190 @@
+import pathlib
+from abc import abstractmethod
+from typing import Dict
+from typing import Optional
+from typing import Sequence
+
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.distributions import BaseDistribution
+from etna.pipeline.base import BasePipeline
+
+
+class BasePredictionIntervals(BasePipeline):
```

Comment:
Why do we actually inherit from `BasePipeline` as `BasePredictionIntervals` is just a wrapper?